### ISSUE TITLE
Alt text on images is not being generated

### ISSUE THREAD

**Issue by agucova** [ForumMagnum/ForumMagnum#6134]
Apparently, alt text added to images of posts (using ckEditor) don't get rendered in the final text.

You can see this [on a post I made recently in the EA Forum](https://forum.effectivealtruism.org/posts/vjyWBnCmXjErAN6sZ/kelsey-piper-s-recent-interview-of-sbf), which included alt text on the images (copied from the Vox article), however no alt text can be found on the HTML of the published post.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/628521446211730/1203397852727642) by [Unito](https://www.unito.io)

**Comment by agucova** [comment]
As far as I could test locally, the problem seems to come from the sanitization function server-side:

https://github.com/ForumMagnum/ForumMagnum/blob/6efaf14e4a1964bd90f82af30b3cce5b35bfad8d/packages/lesswrong/server/vulcan-lib/utils.ts#L32-L35

Adding `alt` to the allowed attributes for `img` tags seems to solve this problem. I wonder if other important accessibility attributes could also be getting filtered.

**Comment by agucova** [comment]
I've made a PR to solve the problem.
